Kbs.Data/User/UserRepository.cs

```diff
@@ -51,6 +49,19 @@ public UserEntity GetByCredentials(string email, string password)
         return user;
     }
 
+    public UserEntity GetByEmail(string email)
+    {
+        var user = _connection.Query<UserEntity>("SELECT * FROM Users WHERE Email = @Email", new { Email = email })
+            .FirstOrDefault();
+
+        if (user == null)
+        {
+            return null;
+        }
+
+        return user;
```

Comment:
Kan gecombineerd worden in 1 statement:
```suggestion
        return user;
```